### PR TITLE
STYLE: Conform to examples modules naming convention.

### PR DESCRIPTION
**Pull request by jhlegarreta**
Conform to examples module naming convention.

### CODE REVIEW

File: examples/CMakeLists.txt [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-    ITKIOJPEG
-    ITKIOPNG
-  )
-include(${ITK_USE_FILE})
-
-add_definitions(-D_USE_MATH_DEFINES=1)
-
-add_executable(DumpSlideInformation DumpSlideInformation.cxx)
-target_link_libraries(DumpSlideInformation ${ITK_LIBRARIES})
+cmake_minimum_required(VERSION 2.8)
```
Comment (hjmjohnson) [inline examples/CMakeLists.txt]: Why not cmake 3.10.2?  Or at least 

cmake_minimum_required(VERSION 2.8...3.13.2)   
See https://cmake.org/cmake/help/v3.13/command/project.html?highlight=project

Comment (jhlegarreta) [reply examples/CMakeLists.txt]: @hjmjohnson Thanks for pointing out ! The reason why all lines seem to have changed is that that very line had a carriage return that I was unable to remove without passing the file through the `dos2unix` tool. But, yes, I also proposed to update the CMake minimum required version in PR #22.

**Comment by jhlegarreta** [comment]
Same error and comment as in PR #22.

**Review by thewtex — APPROVED** [review]

**Comment by jhlegarreta** [comment]
Merging then. The CI issues will be addressed in another PR and the issue #24 has been opened as a reminder.

**Pull request merged**